src/dal/Jalasoft.TeamUp.Projects.DAL/InvitationsInMemoryRepository.cs

```diff
@@ -60,17 +60,22 @@ public IEnumerable<Invitation> GetAllInvitationsByResumeId(int resumeId)
 
         public Invitation GetById(Guid id)
         {
-            throw new NotImplementedException();
+            return invitations.FirstOrDefault(x => Equals(x.Id, id));
         }
 
         public void Remove(Guid id)
         {
             throw new NotImplementedException();
         }
 
-        public Invitation UpdateById(Invitation project)
+        public Invitation UpdateById(Invitation invitation)
         {
-            throw new NotImplementedException();
+            var invitationsList = new List<Invitation>(invitations);
```

Comment:
I think there's no need to use this extra variable, all could be performed in `invitations` only

Reply:
or maybe am I missing something? please double check

Reply:
We use this extra variable to have the methods Remove and Add